assets/js/app.js

```diff
@@ -43,4 +43,6 @@ liveSocket.connect()
 // >> liveSocket.enableDebug()
 // >> liveSocket.enableLatencySim(1000)  // enabled for duration of browser session
 // >> liveSocket.disableLatencySim()
-window.liveSocket = liveSocket
\ No newline at end of file
+window.liveSocket = liveSocket
+
+window.addEventListener("DOMContentLoaded", e => addSampleEventListeners());
```

Comment:
I had some weird issues where when I changed pages only the first few samples worked. The addSampleEventListeners() function currently only seems to run on a full refresh or first load of the page. I did some looking and it seems that the `phx:page-loading-stop` event runs every time the page changes and finishes loading, and it seems to fix this. (https://hexdocs.pm/phoenix_live_view/js-interop.html gives an example of using these phoenix events)

```suggestion
window.addEventListener("phx:page-loading-stop", e => addSampleEventListeners())
```

Reply:
Good catch! that's what I get for not thoroughly testing this after a quick afternoon of hacking it together 😂 👍